src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -203,7 +203,15 @@ public function regenerate($destroy = false, $lifetime = null)
             $this->metadataBag->stampNew();
         }
 
-        return session_regenerate_id($destroy);
+        $isRegenerated = session_regenerate_id($destroy);
+
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
Indeed this is what one of PHP contrinbutors said. I'm waiting for others to express their opinion.

If this is not fixed in older PHP versions, we don't need all the processing done by `loadSession()`.